eclipse-scout-core/test/tree/TreeSpec.ts

```diff
@@ -1278,6 +1278,179 @@ describe('Tree', () => {
       }, node.childNodes);
     });
 
+    it('does check the children if tree is in sync mode', () => {
```

Comment:
you can add a describe block to group all sync mode cases: describe('AutoCheckStyle.SYNC_CHILD_AND_PARENT_STATE)